templates/index.twig

```diff
@@ -3,5 +3,13 @@
 {% block content %}
 <div class="col-md-12">
 	{% include "article-list.twig" with {"wordCount": 100} %}
+	<div class="pagination-buttons">
```

Comment:
You end up with a couple empty divs that make no difference to the page. I just copied this from the news twig file 😛